pipeline.py

```
@@ -1170,7 +1230,121 @@ def _paddleocr_ocr(image) -> OcrResult:
+                lines.append(text)
+        
+        # Join texts
+        full_text = " ".join(texts)
+        
+        # Create result
+        ocr_result = OcrResult(
+            text=full_text,
+            lines=lines,
+            confidences=confidences,
+            engine=engine_name
+        )
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'lines' in constructor call

(E1123)